04.ls/lib/ls_command.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require_relative 'entry'
+require_relative 'formatter'
+
+class LsCommand
+  def initialize(path)
+    path ||= '.'
+    @path = File.expand_path(path)
+  end
+
+  def generate
+    entries = generate_entries
+    Formatter.format(entries)
+  end
+
+  def generate_entries
+    entries = Dir.entries(@path).sort.map { |entry| Entry.new("#{@path}/#{entry}") }
```

Comment:
```suggestion
    entries = Dir.entries(@path).sort.map { |entry| Entry.new(entry) }
```

Entryクラスの実装を見ると`@path`の部分は活用されていないような？

あと、`Entry.new(entry)`は命名が何かおかしいことを示唆してそうです。

https://bootcamp.fjord.jp/pages/class-like-class-and-non-class-like-class#%E3%82%AF%E3%83%A9%E3%82%B9%E5%90%8D%E3%81%A8%E5%90%8C%E5%90%8D%E3%81%AE%E5%A4%89%E6%95%B0%E3%82%92%E7%99%BB%E5%A0%B4%E3%81%95%E3%81%9B%E3%81%AA%E3%81%84